### DISCUSSION TITLE
Slack alerts for database errors?

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#18192] [Ideas]
Hi,

Having to login to the client to check if there are any errors (specifically for us ETL related) means these errors when they occur are usually missed.

Having a slack integration to get real time alerts of errors would be a great feature, any plans to implement something like this?

Thanks

**Comment by ppekrol** [comment]
hi @nickchampion 

we do have some plans to allow direct integrations of our logs with various sources, but at this point of time it is hard for me to give you an ETA.

Have you thought about using logging aggregators such as e.g. grafana loki or cloudwatch and then setting up slack alerts?

**Reply by ayende** [reply]
Alternatively, we would be very happy to accept a pull request for this

**Reply by nickchampion** [reply]
Thanks for the response @ppekrol I'd not considered using log aggregators, are there docs you can point me to for this?

**Reply by ppekrol** [reply]
I think you would have to check directly within the aggregators docs e.g. https://grafana.com/docs/loki/latest/send-data/
